### ISSUE TITLE
i can't able to build apk after updation of flet 0.23.1 

### ISSUE THREAD

**Issue by Acccjifbhi** [flet-dev/flet#3519]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the bug

before flet 0.23.* iam able to build the apk but after update i would not able to build the apk what can do?

### To reproduce

.

### Operating System

Windows

### Operating system details

Windows 10

### Flet version

0.23.*

### Regression

No, it isn't

**Comment by ndonkoHenri** [comment]
What errors do you get? 
Please fill the issue template when reporting issues. We cannot guess what errors you receive!

**Comment by FeodorFitsner** [comment]
It is fixed in Flet 0.23.1.

Please cleanup Cookiecutter cache before running `flet build` command.

On Windows:

```
rd /s /q %USERPROFILE%\.cookiecutters
```

On Mac/Linux:

```
rm -rf ~/.cookiecutters
```
